Dockerfile

```diff
@@ -127,6 +127,13 @@ RUN gcc -DHOMEDIR=\"/data/riotbuild\" -DUSERNAME=\"riotbuild\" /tmp/create_user.
     && chmod u=rws,g=x,o=- /usr/local/bin/create_user \
     && rm /tmp/create_user.c
 
+# Installs the complete ESP8266 toolchain in /opt/esp (170 MB after cleanup) from binaries
+RUN echo 'Adding esp8266 toolchain' >&2 && \
+    cd /opt && \
+    git clone https://github.com/gschorcht/RIOT-Xtensa-ESP8266-toolchain.git esp
```

Comment:
Do we want to use a particular tag or commit or do we just want the latest master commit?

Reply:
yes, this needs to be pinned.

Reply:
@gschorcht can you address the comments from @gebart and @kaspar030 ?

Reply:
@gebart Oh, your question addressed my toolchain repository and not your riotdocker repository. Sorry, I did not realize it. Why not to use simply the last master commit? I don't think that it will change in future. If it does, I could change the docker file then and place a new PR.

Reply:
You need to do it in two steps, first clone, then checkout.